### ISSUE TITLE
[bug] Panic when output doesn't get created

### ISSUE THREAD

**Issue by mrobinsn** [moonrepo/moon#851]
With the following minimal reproduction:

```yaml
$schema: https://moonrepo.dev/schemas/project.json
tasks:
  thing:
    command: touch output
    inputs: []
    outputs:
    - something-else
```

`moon` throws the following error:
```bash
❯ moon run thing
▪▪▪▪ moon-output-panic:thing
▪▪▪▪ moon-output-panic:thing (6ms, 28b752a9)
thread 'tokio-runtime-worker' panicked at 'called `Result::unwrap()` on an `Err` value: StripPrefixError(())', /Users/runner/work/moon/moon/crates/core/runner/src/runner.rs:85:83
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

Tested with 1.4.0 & 1.5.1

**Comment by milesj** [comment]
@mrobinsn Haven't seen this one yet. Does it still happen if you delete `.moon/cache`?

**Comment by mrobinsn** [comment]
Yep, same result with cache blown away.

**Comment by mrobinsn** [comment]
I should mention that while in my minimal reproduction it's obvious that the `command` doesn't create the `output` the task expects, it can be a lot harder to debug when the command is more complicated or it's just a minor typo.

**Comment by milesj** [comment]
Thanks, will have a fix by monday.

**Comment by milesj** [comment]
Fixed in 1.6
